### PR TITLE
inform new release on diff option

### PR DESCRIPTION
**Pull request by kazuminn**
## What did you implement:
In the diff option, also let us know that the package was newly released,

## How can we verify it:

test

## Todos:
You don't have to satisfy all of the following.

- [x] Write tests
- [ ] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Pass the test by `make test`
- [x] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

### CODE REVIEW

File: report/util.go [hunk 1]
```diff
@@ -362,6 +362,25 @@ func getDiffCves(previous, current models.ScanResult) models.VulnInfos {
...
+	for _, h := range preVinfo.AffectedPackages {
+		preNotFixedYet = h.NotFixedYet
+	}
+
+	var curNotFixedYet bool
+	for _, h := range current.AffectedPackages {
+		curNotFixedYet = h.NotFixedYet
+	}
+
+	if (preNotFixedYet == true) && (curNotFixedYet == false) {
```
Comment (kotakanbe) [inline report/util.go]: return preNotFixedYet && !curNotFixedYet

File: report/util.go [hunk 2]
```diff
@@ -362,6 +362,25 @@ func getDiffCves(previous, current models.ScanResult) models.VulnInfos {
 	return updated
 }
 
+func isNewRelease(current models.VulnInfo, previous models.ScanResult) bool {
+	var preNotFixedYet bool
```
Comment (kotakanbe) [inline report/util.go]: func isCveFixed() is better

File: report/util.go [hunk 3]
```diff
@@ -362,6 +362,25 @@ func getDiffCves(previous, current models.ScanResult) models.VulnInfos {
...
+	preVinfo, _ := previous.ScannedCves[current.CveID]
```
Comment (kotakanbe) [inline report/util.go]: This logic of for loop checks only the last entry of prevInfo.AffectedPackages slice.
so please change as follows to check all elements.

```
_ func isNewRelease(current models.VulnInfo, previous models.ScanResult) bool {
      preVinfo, _ := previous.ScannedCves[current.CveID]
+     pre := map[string]bool{}
      for _, h := range preVinfo.AffectedPackages {
~         pre[h.Name] = h.NotFixedYet
      }

~     cur := map[string]bool{}
      for _, h := range current.AffectedPackages {
~         cur[h.Name] = h.NotFixedYet
      }
~_    return !reflect.DeepEqual(pre, cur)
  }
```

File: report/util_test.go [hunk 4]
```diff
@@ -334,3 +334,84 @@ func TestDiff(t *testing.T) {
...
+							CveID:        "CVE-2017-0001",
+							LastModified: time.Time{},
+						},
+					),
+				},
+				prev: models.ScanResult{
+					ScannedCves: models.VulnInfos{
+						"CVE-2017-0001": {
+							CveID:            "CVE-2017-0001",
+							AffectedPackages: models.PackageStatuses{{NotFixedYet: true}},
```
Comment (kotakanbe) [inline report/util_test.go]: {
Name: aaa,
NotFixedYet: true,
}

**Comment by kazuminn** [comment]
refactor!  Done!!

**Comment by kotakanbe** [comment]
@kazuminn Thanks!

**Pull request merged**